contents/handbook/company/diversity.md

```diff
@@ -1,44 +1,64 @@
 
-We view increased inclusion into tech as a wonderful ethical benefit of our policies, but it's not the *fundamental reason* for encouraging diversity at PostHog. Underrepresented great talent is *particularly great talent*, that *any* business should want.
+We are currently trialing this approach and we still have some limitations to overcome:
+* We don’t currently track diversity data as part of the application process. While some characteristics of underrepresentation will be visible in the interview process, others are not. 
+* For some roles (e.g. Full-Stack Engineer), we hire constantly. This makes the Rooney rule a little harder to make meaningful, but we also want to make sure to keep the pipeline as diverse as possible. 
+* Speed is a core PostHog value, and that includes hiring. So we need to work out how to be fast, deliver a great candidate experience, while also doing a better job at diversifying our hiring.  
 
 ## An inclusive place to work
 
-We are always keen to find ways to make the culture at PostHog as inclusive as possible. We are by no means perfect, as with any company, but it's a recurring discussion topic that the team work on improving over time.
+We are always keen to find ways to make the culture at PostHog as inclusive as possible. We are by no means perfect, but we are committed to acting with positive intent and pushing ourselves to improve. 
 
 We don't just state that we care - these are some of the things we've implemented so far:
 
-* [All remote](/handbook/company/team) - so we can hire people from any country in the world. We have people in ~10 countries, with no office.
-* [Asynchronous and transparent communication](/handbook/company/communication) - so people can get the context they need to work effectively across multiple timezones.
+* [All remote](/handbook/company/team) - so we can hire people from any country in the world. We have people in ~10 countries, with no office. If you are unable to work at home, we provide you with $200/month to use on a coworking space of your choice.
+* [Asynchronous and transparent communication](/handbook/company/communication) - so people can get the context they need to work effectively across multiple time zones and on schedules that suit them.
 * [Unlimited vacation policy](/handbook/people/time-off/#permissionless-time-off) with mandatory minimum time off - so you can fit work around your life.
 * Very [generous parental leave](/handbook/people/time-off/#parental-leave) - so those raising families can do so while still working for us.
-* [Top of market pay](/handbook/people/compensation) - to reduce financial stress that often comes with working for startups, or prevents many from even applying.
+* Transparent and equal to [top of market pay](/handbook/people/compensation) - to reduce the financial stress that often comes with working for startups, or prevents many from even applying.
```

Comment:
I'd probably re-word to "very generous" pay. We won't _always_ be right at the top of market, but we want to be very generous at least.